packages/wonder-blocks-data/src/components/__tests__/data.test.js

```diff
@@ -261,22 +262,24 @@ describe("Data", () => {
                     hydrate: true,
                 };
                 const fakeChildrenFn = jest.fn(() => null);
-                const wrapper = mount(
+                const wrapper = render(
                     <Data handler={fakeHandler} options={"options"}>
                         {fakeChildrenFn}
                     </Data>,
                 );
                 // We want to make sure we render the error state so we can
                 // see our switch back to loading.
-                await fulfillSpy.mock.results[0].value;
+                await act(() => fulfillSpy.mock.results[0].value);
```

Comment:
Seeing this got me thinking maybe we should have something similar to trigger GraphQL request fulfilment in tests so that we can wrap that inside an `await act(() => {})` as well instead of having the `await wait(0)` or `await waitFor(() => {...})`.